arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmio/instructions/LimbToRamOneTarget.java

```diff
@@ -19,31 +19,19 @@
 import static net.consensys.linea.zktracer.module.mmio.MmioPatterns.updateTemporaryTargetRam;
 import static net.consensys.linea.zktracer.types.Bytecodes.readLimb;
 
-import lombok.RequiredArgsConstructor;
 import net.consensys.linea.zktracer.module.mmio.MmioData;
 import net.consensys.linea.zktracer.module.mmu.MmuData;
-import net.consensys.linea.zktracer.module.mmu.values.MmuToMmioConstantValues;
-import net.consensys.linea.zktracer.module.mmu.values.MmuToMmioInstruction;
 import net.consensys.linea.zktracer.types.Bytes16;
 
-@RequiredArgsConstructor
-public class LimbToRamOneTarget implements MmioInstruction {
-  private final MmuData mmuData;
+public class LimbToRamOneTarget extends MmioInstruction {
 
-  private final int instructionNumber;
+  public LimbToRamOneTarget(MmuData mmuData, int instructionNumber) {
+    super(mmuData, instructionNumber);
+  }
 
   @Override
   public MmioData execute() {
-    final MmuToMmioConstantValues mmuToMmioConstantValues = mmuData.mmuToMmioConstantValues();
-    final MmuToMmioInstruction mmuToMmioInstruction =
-        mmuData.mmuToMmioInstructions().get(instructionNumber);
-
-    MmioData mmioData =
-        new MmioData(
-            mmuData.hubToMmuValues(),
-            mmuToMmioConstantValues,
-            mmuToMmioInstruction,
-            mmuData.exoSumDecoder());
+    final MmioData mmioData = super.execute();
```

Comment:
This is like in the precompile section I take it ? We have a generic method that runs initially and a more refined method that runs in the respective constructor ?